test/common/testutil.cc

```diff
@@ -76,6 +76,30 @@ pid_t GetParentPid(const pid_t pid) {
   return parent_pid;
 }
 
+std::string GetProcessname(const pid_t pid) {
+#ifdef __APPLE__
+  std::err << "Implementation missing to get process name by pid on MAC\n";
```

Comment:
```suggestion
  std::cerr << "Implementation missing to get process name by pid on MAC\n";
```
On macOS this should actually be easy to get using `proc_pidpath` from `libproc.h`, I can provide the implementation in a future PR.